packages/browser/src/client.ts

```diff
@@ -104,10 +112,34 @@ export class BrowserClient extends BaseClient<BrowserClientOptions> {
    * @inheritDoc
    */
   protected _sendEvent(event: Event): void {
-    const integration = this.getIntegration(Breadcrumbs);
-    if (integration) {
-      integration.addSentryBreadcrumb(event);
+    // We only want to add the sentry event breadcrumb when the user has the breadcrumb integration installed and
+    // activated its`sentry` option.
```

Comment:
```suggestion
    // activated it's `sentry` option.
```

Reply:
while I do now notice that there's a space missing I think this should actually be "its" because "it is" would definitely not fit here